src/main/java/me/totalfreedom/totalfreedommod/command/Command_deop.java

```diff
@@ -30,16 +31,17 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
                 player = onlinePlayer;
             }
         }
-
+        Player target = Bukkit.getPlayer(args[0]);
```

Comment:
you're not checking if the player exists and why do you need a target if the player already exists, an you might be sending the message to an offlineplayer

Reply:
`offlinePlayer` does not allow you send messages so I can't use the `player` variable.